src/components/testCaseList/TestCaseList.tsx

```diff
@@ -257,12 +272,31 @@ const TestCaseList = (props: TestCaseListProps) => {
   };
   const readerString = generateSRString(testCases);
   const executionResultLength = Object.keys(executionResults).length;
+
+  const onTestCaseImport = async (testCases: TestCase[]) => {
+    setImportDialogState({ ...importDialogState, open: false });
+    setLoadingState(() => ({
+      loading: true,
+      message: "Importing Test Cases...",
+    }));
+
+    try {
+      await testCaseService.current.createTestCases(measureId, testCases);
+      retrieveTestCases();
+    } catch (error) {
+      setErrors([...errors, IMPORT_ERROR]);
+    } finally {
+      setLoadingState({ loading: false, message: "" });
+    }
+  };
+
   return (
     <div
       tw="grid lg:grid-cols-6 gap-4 mx-8 my-6 shadow-lg rounded-md border border-slate bg-white"
       style={{ marginTop: 16 }}
     >
-      {!initialLoad && (
+      <span>canEdit: {canEdit ? "yes" : "no"}</span>
```

Comment:
Probably a debug statement?

Reply:
oh crap - yes that is a debug statement from when I was debugging a button problem in Jest...removing immediately..

Reply:
removed in latest commit